pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
+
+- Toggle between play and pause:
+
+`mpc toggle`
+
+- Stop playing:
+
+`mpc stop`
+
+- Show information about currently playing song:
+
+`mpc status`
+
+- Play next song:
+
+`mpc next`
+
+- Play previous song:
+
+`mpc prev`
+
+- Seek playing song:
```

Comment:
Seek is a bit of a technical term, albeit the correct one of course. I wonder if "jump forward/backward within the playing song", or something to that effect ("move"?), be a better option. What do you think?